### ISSUE TITLE
Should common-panel-set element have selectedIndex?

### ISSUE THREAD

**Issue by bkardell** [bkardell/common-panel#5]
It seems like an obvious sort of add, select boxes (another kind of single-select mechanism) have this, as do most implementations of tabs, decks and accordions.  If so it'd be reflected so you could do

myPanelSet.selectedIndex = 4;

And it would update the attribute and do the right thing.  I didn't start with that because there are numerous cases where that sort of API is clunkier than dealing with element references you'd get another way, but I can definitely see it's utility.

**Comment by LJWatson** [comment]
The idea being that you could select a specific panel to be displayed by default? Makes sense to me.

**Comment by bkardell** [comment]
@LjWatson no, you can do that in markup with the expansion state today and using an index to set via markup is markedly harder... the idea being that you could read this attribute or read/write the runtime javascript property to say "select the third one".  We basically have to track this internally anyway and that's _functionally_ how it's managed, so exposing it seems kind of non-harmful and I can imagine some cases where it could be useful.
